src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.tt

```diff
@@ -116,31 +80,26 @@ foreach (XElement verb in commands.Descendants(XName.Get("Verb","")))
                 {
 <#
         }
+
+        // Look for options directly under each noun
         foreach (XElement option in noun.Descendants())
         {
-            if (IsArgumentOrOption(GetProperty(option)))
+            var data = new OptionData(option);
+            if (data.BaseClass == "Argument")
             {
-                string optionName = option.Attribute(XName.Get("Name", ""))?.Value;
-                string optionLongName = option.Attribute(XName.Get("LongName", ""))?.Value;
-                string optionHelp = option.Attribute(XName.Get("Help", ""))?.Value;
-                string optionFormalName = optionName.Replace("-","");
-                if (IsArgument(GetProperty(option)))
-                {
 #>
-                    CommandArgument <#= optionFormalName #> = <#= nounFormalName #>Cmd.Argument(
-                        "<#= optionLongName != null ? optionLongName : optionName #>", <#= optionHelp != null ? "Strings."+optionHelp : "" #>);
+                    CommandArgument <#= data.FormalName #> = <#= nounFormalName #>Cmd.Argument(
+                        "<#= data.LongNameOrName #>", <#= data.Help #>);
```

Comment:
```suggestion
                CommandArgument <#= data.FormalName #> = <#= commandFormalName #>Cmd.Argument(
                    "<#= data.LongNameOrName #>",
                    <#= data.Help #>,
                    multipleValues: <#= data.MultipleValues #>);
```
Adding support for multiple values under `Argument` element. [Reference](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PushCommand.cs#L66)

Reply:
Why don't you do a separate PR for this MultipleValues feature that you want to add for arguments (and other improvements you decide to make). You can do that as part of your using the feature.
If I add it to this PR, the feature isn't even being used yet, or tested in any way.